app/models/mixins/supports_feature_mixin.rb

```diff
@@ -73,6 +73,7 @@ module SupportsFeatureMixin
     :create_host_aggregate      => 'Host Aggregate Creation',
     :create_network_router      => 'Network Router Creation',
     :create_security_group      => 'Security Group Creation',
+    :create_snapshot            => 'Create Snapshot',
```

Comment:
there is already `:snapshot_create` down there. @gauravaradhye can we use this instead of `:create_snapshot`?